charts/worker/templates/tests/_test_helpers.tpl

```diff
@@ -103,7 +118,9 @@ endpoints:
 - name: helloworld
   defined:
     name: Greeter # Name could be omitted if equal to endpoint's name
-    path: "/usr/local/lib/skyramp/idl/grpc/files/helloworld.proto"
+    path: "helloworld.proto"
+    protoPaths:
+      - /usr/local/lib/skyramp/idl/grpc/files/pb
```

Comment:
is this consistent with `skyramp`? I remember you mentioned only singular is accepted

Reply:
good catch, I had this before I made the changes